src/main/resources/pathToContinentsFile.properties

```diff
@@ -0,0 +1,6 @@
+source.pathToAfrica = D:/java/TimeInCapitals/africa_capitals.txt
```

Comment:
Proponowałbym zmienić nazwę parametru na `paths.africa` i analogicznie dla innych kontynentów, będzie moim zdaniem czytelniej.

Reply:
Poprawione, również dla pozostałych.

Reply:
Nie widzę żadnej zmiany?

Reply:
hmm, na delevelopie jest zmiana, sprawdzałem przed chwilą.

Reply:
To przenieśmy dyskusję na develop